src/frontend/src/lib/components/icons/OisyWalletLogo.svelte

```diff
@@ -1,51 +1,27 @@
-	/>
-	<path
-		fill="url(#b)"
-		class="logo__symbol"
-		d="m18.63 5.49-.69.3v4.22a8.4 8.4 0 0 1 .7 0V5.5Zm-2.3 1.19L17 10.1c-.23.04-.46.08-.68.14l-.62-3.13c.2-.15.4-.29.62-.42Zm-1.88 1.46.99 2.37a8.52 8.52 0 0 0-.64.26l-.88-2.11c.18-.18.35-.36.53-.52Zm-1.47 1.58 1 1.5c-.2.11-.4.24-.58.38l-.86-1.29.44-.59Zm-1.12 1.65c-.12.2-.23.42-.34.63l.67.68.49-.49-.82-.82Zm-.82 1.66.56.37a8.3 8.3 0 0 0-.38.57l-.43-.28.25-.66Zm-.56 1.65-.16.68.19.08a7.92 7.92 0 0 1 .26-.64l-.29-.12Zm-.33 1.64.09.01a8.25 8.25 0 0 0-.14.68h-.04c.03-.24.05-.47.09-.7Zm-.14 1.62a14.14 14.14 0 0 0 0 .69 8.4 8.4 0 0 1 0-.69Zm.05 1.63h.04c.04.22.08.45.14.67l-.1.02-.08-.7Zm.26 1.64.16.68.3-.12a8.21 8.21 0 0 1-.27-.64l-.2.08Zm.47 1.67.42-.28c.12.2.25.38.39.57l-.56.37a13.81 13.81 0 0 1-.25-.66Zm.73 1.69c.1.21.22.42.34.63l.82-.82a8.35 8.35 0 0 1-.49-.49l-.67.68Zm1.02 1.69.86-1.29c.19.14.38.27.58.39l-1 1.49a13.79 13.79 0 0 1-.44-.59Zm1.38 1.66.54.52.98-2.38a8.24 8.24 0 0 1-.64-.26l-.88 2.12Zm1.79 1.54.62-3.13c.23.06.45.1.68.14l-.68 3.42a13.84 13.84 0 0 1-.62-.43Zm2.23 1.32.7.3v-4.52a8.4 8.4 0 0 1-.7 0v4.22Zm2.66.93-1.04-5.24.68-.13 1.1 5.53a13.68 13.68 0 0 1-.74-.16Zm3.03.38a15.28 15.28 0 0 0 .74-.02l-2.6-6.27a8.23 8.23 0 0 1-.64.26l2.5 6.03Zm3.23-.34-4.26-6.4a8.32 8.32 0 0 0 .57-.38l4.4 6.6-.7.18Zm1.3 1.93.57-.38-.57.38Zm2.81-2.22.49-.49-.49.49Zm-.88-.88.63-.35-6.34-6.34a8.32 8.32 0 0 1-.49.49l6.2 6.2Zm2.94-2.03-8.06-5.38.39-.57 8.18 5.47a14.75 14.75 0 0 1-.51.48Zm.27.18.38-.57-.38.57Zm-7.5-6.96 9.25 3.83.26-.64-9.25-3.83c-.08.22-.16.43-.26.64Zm10.35.43-9.81-1.96c.05-.22.1-.45.13-.68l9.82 1.96-.14.67Zm-9.59-3.57h10.01v-.69H26.56a8.4 8.4 0 0 1 0 .69Zm9.73-3.57L26.47 17a8.26 8.26 0 0 0-.13-.68l9.81-1.95.14.68Zm-10.23.38 9.25-3.83-.26-.64-9.25 3.83c.1.2.18.42.26.64Zm7.48-6.94-8.18 5.48a8.25 8.25 0 0 0-.39-.58l8.06-5.38c.17.16.35.32.51.49Zm.14-.08-.38-.58.38.58ZM30.09 6l-6.2 6.2c.17.15.33.31.49.48l6.34-6.34a13.85 13.85 0 0 0-.63-.35Zm-2.52-1-4.4 6.6a8.33 8.33 0 0 0-.57-.39l4.26-6.39a15 15 0 0 1 .71.18Zm1.16-1.73-.57-.38.57.38ZM25.6 1.52l-.64-.26.64.27ZM24.37 4.5l-2.6 6.27a8.23 8.23 0 0 0-.64-.26l2.5-6.02a14 14 0 0 1 .74 0Zm-3.03.2-1.1 5.54a8.22 8.22 0 0 0-.68-.14l1.04-5.24.74-.16ZM22.2.42l-.68-.14.67.14Z"
-	/>
-	<g fill="currentColor" fill-rule="evenodd" class={hideName ? 'hidden md:flex' : 'flex'}>
-		<path d="m99.51 10.7 4.45 5.46 4.45-5.45h7.04l-8.53 10v5.2h-5.93v-5.2l-8.53-10h7.05Z" />
+<div class="mr-3">
+	<svg width="44" height="44" viewBox="0 0 44 44" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
correct: i split the logo into the image (the effective logo) and the written part: that one is the image that is constant

shall I separate it? rename the entire component maybe? open to suggestions

Reply:
Since we are already there I would just use two svgs and have no svg code within the component - i.e. I would not keep the constant part in the component.

if small this svg else this svg.

But no strong opinion neither.

Reply:
then what if the image part (the real logo), i put it in a separate SVG? we would have 3 SVGs (1 fixed and 2 changing), but maybe that logo will come in handy at some point, no?

Reply:
I would have two with the logo duplicated. Svg are easy to generate from figma. But like I said no strong opinion, fine with current implementatin.